src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -45,24 +46,23 @@ public GitHubPackage(IServiceProvider serviceProvider)
             this.serviceProvider = serviceProvider;
         }
 
-        protected override void Initialize()
+        protected override async tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
-            base.Initialize();
-            IncrementLaunchCount();
+            var usageTracker = await GetServiceAsync(typeof(IUsageTracker)) as IUsageTracker;
+            await tasks.Task.Run(() => usageTracker.IncrementLaunchCount());
```

Comment:
What's the point of the Task.Run when `AsyncPackage.InitializeAsync` already starts on a background thread? (is that true, @BertanAygun?)